tutorials/preparing-the-vm.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: page
+title: Preparing the Virtual Machine
+---
+
+Complete the following steps to run Kite example code on a Cloudera Quickstart VM.
+
+* Install a VirtualBox or VMWare [Cloudera QuickStart VM][getvm] version 5.2 or later.
```

Comment:
If we say we recommend one over the other, even if it's true, it calls into question the quality of our VM and ruffles the feathers of a partner. It's fine to say that if one doesn't work, try the other, which covers both bases.